exp/views/responses.py

```diff
@@ -69,6 +69,21 @@ def csv_filename(study: Study, *args) -> Text:
     args = map(str, args)
     return f"{study_name_for_files(study.name)}_{'_'.join(args)}.csv"
 
+def keyword_filename(keywords: object, filetype: str) -> Text:
+    """Generates filename for CSV using keyword key-value pairs.
+
+    Args:
+        keywords (object): Dict with key-value pairs respresenting keywords
+        filetype (str): string indication file extension
+
+    Returns:
+        Text: CSV filename with keyword formatting
+    """
+    return_str = ""
+    for key,value in keywords.items():
+        return_str += f"{key}-{value}_"
+    return f"{return_str}data.{filetype}"
```

Comment:
Could we use `join` instead?  See this [stackoverflow](https://stackoverflow.com/a/3055541) for why is might be a good idea.